tests/graphics/sceneviewer.cpp

```diff
@@ -32,74 +32,148 @@ TEST(cmzn_sceneviewer_api, destroy_context_before_scene_viewer)
 	 cmzn_sceneviewer_destroy(&sv);
 }
 
-TEST(cmzn_sceneviewer_api, set_background_invalid_args)
+TEST(cmzn_sceneviewer_api, background_colour)
 {
-	ZincTestSetup z;
+	ZincTestSetup zinc;
+	const double tolerance = 1.0E-12;
+	const double colourIn3[] = { 0.2, 0.4, 0.6 };
+	const double colourIn4[] = { 0.25, 0.45, 0.65, 0.85 };
+	double colour3[3], colour4[4];
 
-	cmzn_sceneviewermodule_id svm = cmzn_context_get_sceneviewermodule(z.context);
+	cmzn_sceneviewermodule_id svm = cmzn_context_get_sceneviewermodule(zinc.context);
 	cmzn_sceneviewer_id sv = cmzn_sceneviewermodule_create_sceneviewer(svm, CMZN_SCENEVIEWER_BUFFERING_MODE_DEFAULT, CMZN_SCENEVIEWER_STEREO_MODE_DEFAULT);
-
+	EXPECT_NE(static_cast<cmzn_sceneviewer_id>(0), sv);
+
+	// test defaults
+	EXPECT_EQ(CMZN_RESULT_OK, cmzn_sceneviewer_get_background_colour_rgba(sv, colour4));
+	EXPECT_NEAR(0.0, colour4[0], tolerance);
```

Comment:
When I started I wasn't sure what the colour precision was, but it is in fact double so I'll change to EXPECT_DOUBLE_EQ since it reads much better.